src/create_disk.sh

```diff
@@ -208,7 +208,7 @@ if [ "${remote_name}" != NONE ]; then
     remote_arg="--remote=${remote_name}"
     deploy_ref="${remote_name}:${ref}"
 fi
-ostree pull-local "$ostree" "$ref" --repo rootfs/ostree/repo $remote_arg
+time ostree pull-local "$ostree" "$ref" --repo rootfs/ostree/repo $remote_arg
```

Comment:
Leftover?

Reply:
Yeah but I decided to keep it since why not?